### PR TITLE
fail on console.error

### LINKED ISSUE #255: Tests have a lot of errors not failing the test suite
An example:
image

I will make Jest fail on every console.error called to catched those errors.

### PR DESCRIPTION
**Pull request by faboweb**
PR to fix #255 

Help wanted :)

### CODE REVIEW

**Comment by nylira** [comment]
I tried working on these tests a little earlier today but I couldn't get them to pass. Need more time or insight to debug.

**Comment by mappum** [comment]
@nylira I did a git bisect, found that the first bad commit is
2492fa6e777602abadc4aa9fc2203a9112c4212e.

**Comment by nylira** [comment]
@mappum thanks Matt, managed to resolve the issue with that commit id. I should learn how to use git bisect to narrow down the offender.

File: test/unit/helpers/node_mock.js [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
   },
+  coinTxs: () => Promise.resolve({}),
+  candidates: () => Promise.resolve({data: []}),
   postTx: () => Promise.resolve({
     check_tx: { code: 0 },
     deliver_tx: { code: 0 }
   }),
-  sign: () => Promise.resolve(null)
+  sign: () => Promise.resolve(null),
+
```
Comment (jolesbi) [inline test/unit/helpers/node_mock.js]: line break?

**Comment by mappum** [comment]
@faboweb @nylira nice changes! The tests are now passing without issues

File: test/unit/helpers/node_mock.js (hunk 1 above)
Comment (faboweb) [reply test/unit/helpers/node_mock.js]: it was to structure rest endpoints away from the rpc object

**Pull request merged**